Description: correct misleadings in man page
Author: Jörg Frings-Fürst <debian@jff-webhosting.net>
Forwarded: not-needed
Last-Update: 2014-10-23
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: trunk/doc/xsane.man
===================================================================
--- trunk.orig/doc/xsane.man
+++ trunk/doc/xsane.man
@@ -203,8 +203,10 @@ When the flag
 or
 .B -N
 is given then xsane reads the next option as default image filename. The name should be
-of the format "name-###.ext". The selection box for filenames is disabled. This
-option normally should be used with the option
+of the format "name-001.ext". (or any other number instead of 001) that 001 part
+will be used as a filename counter if xsane is so configured. Beware that the
+number of digits must correspond to the configuration of xsane. The selection box
+for filenames is disabled. This option normally should be used with the option
 .B --no-mode-selection
 and
 .B --save.
